### ISSUE TITLE
ident behavior change when upgrading to 0.46.0

### ISSUE THREAD

**Issue by vanniktech** [pinterest/ktlint#1510]
This is my editorconfig file:

```
[*.{kt,kts}]
indent_size=2
continuation_indent_size=2
```

and this is my snippet:

```kotlin
  private val locale: Locale by option(help = "Locale")
    .convert { Locale(it) }
    .default(Locale.getDefault())
```

When running ktlint with 0.45.2 all is fine. However when running `ktlintFormat` with `0.46.0`, the spacings change:

```diff
  private val locale: Locale by option(help = "Locale")
-   .convert { Locale(it) }
+     .convert { Locale(it) }
-   .default(Locale.getDefault())
+     .default(Locale.getDefault())
```

The continuation ident seems to be changing (from 4 to 6). Is this change wanted? Looking at my editorconfig file, the previous behavior seems to be correct.

**Comment by shashachu** [comment]
@vanniktech can you run lint with the `--verbose` flag to confirm which rule it is that's making that change? Unfortunately a number of rules can affect indentation.

**Comment by paul-dingemans** [comment]
It is indeed a regression bug. Output with snippet above in a file called `Foo.kt`:
```
src/main/kotlin/Foo.kt:2:1: Unexpected indentation (2) (should be 4) (indent)
src/main/kotlin/Foo.kt:3:1: Unexpected indentation (2) (should be 4) (indent)
```
